ruby/vampires.rb

```diff
@@ -1,3 +1,4 @@
+# How many people to survey

```

Comment:
For age check line 69

When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`